### PR TITLE
Add gazebo ros pkg to get entity state from ros2 services

### PR DESCRIPTION
**Pull request by elector102**
# 🎉 New feature

## Summary
this PR set the option to use custom world. In that case, I added an empty work with gazebo ros state pkg in order to get entity state from ros2 services.

A good use of that plugin in the world is to use the ground trust position, in order to verify is the robot is where it thinks it is. It is very useful to test and check if the robot have the behavior that we expected. 

## Test it
a way to test that is checking the following service when you run andino_one_robot:
`ros2 service call /get_entity_state gazebo_msgs/srv/GetEntityState '{name: "andino"}'`

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: andino_gazebo/launch/andino_one_robot.launch.py [hunk 1]
```diff
@@ -56,9 +56,15 @@ def generate_launch_description():
...
     )
+
+    # Set the path to the world file
+    world_file_name = 'empty.world'
+
     world_argument = DeclareLaunchArgument(
-        'world', default_value=['empty_world.world'], description='SDF world file'
-    )
+        name='world',
+        default_value=world_file_name,
```
Comment (agalbachicar) [inline andino_gazebo/launch/andino_one_robot.launch.py:65]:
```suggestion
        default_value='empty.world',
```

File: andino_gazebo/launch/andino_one_robot.launch.py [hunk 2]
```diff
@@ -56,9 +56,15 @@ def generate_launch_description():
         default_value='true',
         description='Use simulation (Gazebo) clock if true',
     )
+
+    # Set the path to the world file
+    world_file_name = 'empty.world'
+
```
Comment (agalbachicar) [inline andino_gazebo/launch/andino_one_robot.launch.py:62]:
```suggestion
```

Comment (agalbachicar) [reply andino_gazebo/launch/andino_one_robot.launch.py:62]: To save vertical space, you can simply add the name of the file where it needs to be defined. It is not used anywhere else.

**Review by agalbachicar — CHANGES_REQUESTED** [review]
It works for me. Minor style related comments only.
Not an expert on the env-hooks so I'll defer that to @francocipollone

File: andino_gazebo/worlds/empty.world [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="default">
+    <plugin name="gazebo_ros_state" filename="libgazebo_ros_state.so">
+      <ros>
+      </ros>
+
+      <update_rate>1.0</update_rate>
```
Comment (francocipollone) [inline andino_gazebo/worlds/empty.world:8]: Just double checking, is it ok 1hz for your application? @elector102 @agalbachicar

File: andino_gazebo/worlds/empty.world [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <!-- A ground plane -->
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+  </world>
+
+
```
Comment (francocipollone) [inline andino_gazebo/worlds/empty.world:20]: nit: remove unnecessary spaces

**Review by francocipollone — CHANGES_REQUESTED** [review]
lgtm! Some minor changes

1. Just for reference I assume you are pushing for this change because the necessity of having the `gazebo_ros_state` plugin. Just for the sake of documentation can you mention which is your use case?

2. We could add in a following PR some obstacles to the empty world (3 or 4 primitives) just to make it a bit more enjoyable.

File: andino_gazebo/worlds/empty.world (hunk 3 above)
Comment (elector102) [reply andino_gazebo/worlds/empty.world:8]: Thanks! I will increase that. I think 10 hz for physics things are enough.

File: andino_gazebo/worlds/empty.world (hunk 4 above)
Comment (elector102) [reply andino_gazebo/worlds/empty.world]: Fixed!

File: andino_gazebo/worlds/empty.world (hunk 3 above)
Comment (elector102) [reply andino_gazebo/worlds/empty.world]: Fixed!

File: andino_gazebo/env-hooks/andino_gazebo.sh.in [hunk 5]
```diff
@@ -0,0 +1 @@
+ament_prepend_unique_value GAZEBO_RESOURCE_PATH "$AMENT_CURRENT_PREFIX/share/@PROJECT_NAME@/worlds"
```
[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/env-hooks/andino_gazebo.sh.in]: Fixed!

File: andino_gazebo/env-hooks/andino_gazebo.dsv.in [hunk 6]
```diff
@@ -0,0 +1 @@
+prepend-non-duplicate;GAZEBO_RESOURCE_PATH;share/@PROJECT_NAME@/worlds
```
[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/env-hooks/andino_gazebo.dsv.in]: Fixed!

File: andino_gazebo/launch/andino_one_robot.launch.py (hunk 2 above)
Comment (elector102) [reply andino_gazebo/launch/andino_one_robot.launch.py]: Fixed!

File: andino_gazebo/launch/andino_one_robot.launch.py [hunk 7]
```diff
@@ -75,11 +81,15 @@ def generate_launch_description():
...
     )
+
     # Gazebo launch
-    gazebo = IncludeLaunchDescription(
+    include_gazebo = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
-            os.path.join(pkg_gazebo_ros, 'launch', 'gazebo.launch.py')
-        )
+            os.path.join(pkg_gazebo_ros, 'launch', 'gazebo.launch.py'),
+        ),
```
[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/launch/andino_one_robot.launch.py:86]: What change? The `,` I need for the launch argument on the line 87, 85 I can revert.

[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/launch/andino_one_robot.launch.py:86]: Fixed!

File: andino_gazebo/launch/andino_one_robot.launch.py [hunk 8]
```diff
@@ -75,11 +81,15 @@ def generate_launch_description():
             os.path.join(pkg_andino_gazebo, 'launch', 'spawn_robot.launch.py')
         ),
     )
+
     # Gazebo launch
-    gazebo = IncludeLaunchDescription(
+    include_gazebo = IncludeLaunchDescription(
```
[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/launch/andino_one_robot.launch.py]: Fixed

File: andino_gazebo/worlds/empty.world [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+</sdf>
```
[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/worlds/empty.world]: Fixed

**Comment by elector102** [comment]
@agalbachicar @francocipollone ! Thanks for your comments! I addressed that !

File: andino_gazebo/worlds/empty.world (hunk 3 above)
Comment (francocipollone) [reply andino_gazebo/worlds/empty.world]: Note that this is just update rate of the plugin. If it happen you want to modify the physics's rate you need to add other tags.

**Review by francocipollone — APPROVED** [review]
Approved! Please take a look at the comments I left.

File: andino_gazebo/launch/andino_one_robot.launch.py [hunk 10]
```diff
@@ -83,9 +80,9 @@ def generate_launch_description():
     )
 
     # Gazebo launch
-    include_gazebo = IncludeLaunchDescription(
+    gazebo = IncludeLaunchDescription(
```
[the remark this answers is not in the document]
Comment (elector102) [reply andino_gazebo/launch/andino_one_robot.launch.py:83]: I get back to the original name, sorry, I didn't rename it.

**Comment by elector102** [comment]
@agalbachicar I made some changes that you suggested! I need your review to merge that PR. Thanks.

File: andino_gazebo/worlds/empty.world (hunk 3 above)
Comment (agalbachicar) [reply andino_gazebo/worlds/empty.world]: 10Hz is fine. Thanks for catching it.

**Review by agalbachicar — APPROVED** [review]
LGTM

**Pull request merged**